evm/chains/base.py

```diff
@@ -115,7 +136,9 @@ def get_block_header_by_hash(self, block_hash):
         raise NotImplementedError("Chain classes must implement this method")
 
     @abstractmethod
-    def create_header_from_parent(self, parent_header, **header_params):
+    def create_header_from_parent(self,
+                                  parent_header: BlockHeader,
+                                  **header_params: Dict) -> BlockHeader:
```

Comment:
Same here, can we get rid of the *implicit* `Any` for the `Dict` by *explicitly* setting the types.